cnf-tests/Dockerfile.openshift

```
@@ -70,6 +70,10 @@ FROM quay.io/openshift/origin-oc-rpms:4.16 AS oc
 # Final image
 FROM openshift/origin-base
 
+ENV IMAGE_REGISTRY=quay.io/openshift-kni/
+ENV CNF_TESTS_IMAGE=cnf-tests:4.18
+ENV DPDK_TESTS_IMAGE=dpdk:4.18
+
```

I know we are locking to 4.18 here but might be better to define a single environment variable `OCP_VERSION`  and then populate the versions here based on that here.

```suggestion

ENV OCP_VERSION 4.18
ENV IMAGE_REGISTRY=quay.io/openshift-kni/
ENV CNF_TESTS_IMAGE=cnf-tests:${OCP_VERSION}}
ENV DPDK_TESTS_IMAGE=dpdk:${OCP_VERSION}}

```

---

I added these recommendation [here](https://github.com/openshift-kni/cnf-features-deploy/pull/2187/files) for the master branch

---

Sure, makes sense. Thanks!

---

@swatisehgal Thanks for your review.